### ISSUE TITLE
Feature: place helmet tags in specific place in head

### ISSUE THREAD

**Issue by maliyshock** [nfl/react-helmet#514]
**Do you want to request a *feature* or report a *bug*?**
Feature

**What is the current behavior?**
All tags in helmet appendchilded in the head section, so it means that they come last before closing head tag. 

**What is the expected behavior?**
I need to have a possibility to add them before google analytics or else. For example, prepend them, or specify somehow place in head section, where helmet would add them

**Comment by DanSallau** [comment]
@maliyshock have you found the solution for this ? Currently stucked with this .

**Comment by maliyshock** [comment]
No. This is a feature request, I think there are no possibilities to do that "inside of the box"

You can try to modify the library by itself somewhere here https://monosnap.com/file/7EbKb69GUg4SSq9ZvXulpF39JBYYFM

For example. You can insert your tags instead of appendChild it
https://stackoverflow.com/questions/4793604/how-to-insert-an-element-after-another-element-in-javascript-without-using-a-lib

But the modification of the external library is not the best idea, that's why I am asking for this feature.

**Comment by ghost** [comment]
If you just want to prepend them to the very top of `<head>` then it's simple to do that manually.  

```js
useEffect(() => {
    // react-helmet does not prepend so need to do it manually
    const meta = document.createElement('meta');
    meta.setAttribute('name', 'some-meta-tag-name');
    meta.setAttribute('value', 'some meta tag value');
    const head = document.querySelector('head');
    head.insertBefore(meta, head.firstChild);
  }, []);
```

Would be nice if there was a prop that handled this 
```jsx
<Helmet prepend>
  <meta name="blabla" value="something something" />
</Helmet>
```

if you dont support IE then it's easy to prepend multiple with [ParentNode.prepend](https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/prepend), otherwise looping through and using insertBefore.

Placing them in specific location is different beast. Probably can be done but also seems like more of a headache to deal with

### update 
Just a quick warning about the solution above:

For my case this did not end up working out for me because I was adding a verification code to my site and the vendor just curls and parses the HTML so it needs to be in the initial payload and not injected by JS later.  I switched back to `react-helmet` because I'm using GatsbyJS where the static page rendering had my meta tag in the generated `index.html`

So be aware of possible race conditions if you are injecting tags that are needed by other libraries.
